### PR TITLE
Added POST Audio URL action

### PR DESCRIPTION
**Pull request by mjabali**
Refactored POST Video URL 
Added Trackers prop

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/symbl_ai/actions/post-audio-url/post-audio-url.mjs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            detectEntities: this.detectEntities,
+            languageCode: this.languageCode,
+            mode: this.mode,
+            enableSeparateRecognitionPerChannel: this.enableSeparateRecognitionPerChannel,
+            enableAllTrackers: this.enableAllTrackers,
+            enableSpeakerDiarization: this.enableSpeakerDiarization,
+            diarizationSpeakerCount: this.diarizationSpeakerCount,
+            trackers: JSON.parse(this.trackers),
+          },
+        });
```
Comment (andrewjschuang) [inline components/symbl_ai/actions/post-audio-url/post-audio-url.mjs:122]: **Lint**

Decrease indentation by 2 spaces

File: components/symbl_ai/actions/post-audio-url/post-audio-url.mjs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      label: "Audio URL",
+      description: "The URL of the audio file to be processed.",
+      optional: false,
+    },
+    meetingName: {
+      type: "string",
+      label: "Meeting Name",
+      description: "The meeting name. The default name is set to the conversationId.",
+      optional: true,
+    },
```
Comment (andrewjschuang) [inline components/symbl_ai/actions/post-audio-url/post-audio-url.mjs:23]: Reused `props` can be defined inside app's `propDefinitions`.
This applies to other `props` as well.

See https://github.com/PipedreamHQ/pipedream/pull/2485#discussion_r835346771

**Review by andrewjschuang — CHANGES_REQUESTED** [review]
Thanks @mjabali for this submission!

I left minor a comment about indentation and `propDefinitions`, but other than that it looks good!

File: components/symbl_ai/actions/post-audio-url/post-audio-url.mjs (hunk 1 above)
Comment (mjabali) [reply components/symbl_ai/actions/post-audio-url/post-audio-url.mjs]: Fixed in 8c9deda4.

**Review by andrewjschuang — APPROVED** [review]

**Comment by dannyroosevelt** [comment]
@andrewjschuang feel free to merge

**Comment by mjabali** [comment]
I'm sorry @dannyroosevelt and @andrewjschuang but had to fix some merge conflicts. I'd appreciate your help pushing this one.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**